skopt/gp_opt.py

```diff
@@ -91,16 +91,14 @@ def gp_minimize(func, dimensions, base_estimator=None, acq="LCB", xi=0.01,
         lbfgs is run for 10 iterations optimizing the acquisition function
         over the Gaussian prior.
 
-    * `maxiter` [int, default=1000]:
-        Number of iterations to find the minimum. Note that `n_start`
-        iterations are effectively discounted, such that total number of
-        function evaluations is at most `maxiter`.
+    *  `n_calls` [int, default=100]:
+        Number of times the expensive function is called.
```

Comment:
I think we should add the long version of the explanation, detailing the interplay with `n_random_starts`.